### ISSUE TITLE
randWord array limited to 72

### ISSUE THREAD

**Issue by rodrigofeijao** [ngneat/falso#180]
### Is this a regression?

No

### Description

When getting an array with a given "length" this is limited to the number of items in the seed file (72 in this case).
I've also couldn't find any tests to this function.

### Do you want to create a pull request?

No

**Comment by theryansmee** [comment]
@rodrigofeijao I'm not sure if i've misunderstood you but I have tried passing a length value larger than the number of items in the JSON array and all seems to work as expected _(See screenshot)_
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/7142779/153767019-1352303f-6ac3-4d00-ba0d-45a18ba67e44.png">

**Comment by NetanelBasal** [comment]
@rodrigofeijao can you share a code example with the issue, please?

**Comment by rodrigofeijao** [comment]
Sorry @theryansmee and @NetanelBasal this might be my fault! I think I've totally overlooked an issue I was having when doing the `randJSON()` and assumed the error were in the `randWord` function instead.

Basically I was getting a length mismatch error at this function: https://github.com/ngneat/falso/blob/main/packages/falso/src/tests/rand-json.spec.ts#L22-L29
when using randomized numbers (usually bigger than 10_000). Looking at it again, it seems the problem was just mine and the logic I've created to generate the object keys. 

Here, https://github.com/ngneat/falso/blob/main/packages/falso/src/lib/rand-json.ts#L38, I've started by using `randWord` to generate the array, but as the words are based on a library of 72 elements, the JSON were only getting 72 keys, just because the `reduce()` was overwriting the previous keys. 

I'm creating a PR to fix this issue, and to also improve the performance when length values are super high.
